components/camel-aws/camel-aws2-kinesis/src/main/java/org/apache/camel/component/aws2/kinesis/Kinesis2Configuration.java

```diff
@@ -67,6 +67,8 @@ public class Kinesis2Configuration implements Cloneable {
     private Integer proxyPort;
     @UriParam(defaultValue = "false", description = "If we want to trust all certificates in case of overriding the endpoint")
     private boolean trustAllCertificates;
+    @UriParam(defaultValue = "false", description = "If we want to a KinesisAsyncClient instance set it to true")
```

Comment:
This is producer only, so it needs to be labelled

Reply:
My aim was to make it bi-directional, both consumer and producer will take effect by async connection ?

Reply:
Let me declare a label for consumer then  for the time being